src/main/scala/cromwell/engine/package.scala

```diff
@@ -175,7 +175,7 @@ package object engine {
 
   case class SymbolStoreKey(scope: String, name: String, index: Option[Int], input: Boolean)
 
-  case class SymbolStoreEntry(key: SymbolStoreKey, wdlType: WdlType, wdlValue: Option[WdlValue]) {
+  case class SymbolStoreEntry(key: SymbolStoreKey, wdlType: WdlType, wdlValue: Option[WdlValue], isWorkflowOutput: Boolean) {
```

Comment:
I'm just curious how you'd structure this with an ADT because I don't see it (though I haven't read all of his code yet)...

Also perhaps this could use a default value?